nibabel/cifti2/tests/test_new_cifti2.py

```diff
@@ -386,11 +393,12 @@ def test_plabel():
     hdr = ci.Cifti2Header(matrix)
     data = np.random.randn(2, 3)
     img = ci.Cifti2Image(data, hdr)
-    img.nifti_header.set_intent('NIFTI_INTENT_CONNECTIVITY_UNKNOWN')
 
     with InTemporaryDirectory():
         ci.save(img, 'test.plabel.nii')
         img2 = ci.load('test.plabel.nii')
+        assert_true(img.nifti_header.get_intent()[0]
+                    == 'dense fiber/fan samples')
```

Comment:
shouldn't this be unknown?

Reply:
I matched it with what's consistent in the intent codes
https://github.com/nipy/nibabel/blob/e48b746af1841aed2ff862d1eea8471f35265b2c/nibabel/cifti2/parse_cifti2.py#L55

Reply:
`assert_equal`

Reply:
There is something incorrect here (maybe more than one thing).  Dense fiber samples and dense fan samples are just dscalar files with a specific number and meaning of maps - they use the dscalar intent code and name.  Their extensions are .dfibersamp.nii and .dfansamp.nii.  The dense fans file type is another instance of this.

parcel by label files (which we use the extension .plabel.nii for) are not a "standard" mapping combination, so they should use intent 3000, and intent name "ConnUnknown".